### PR TITLE
fix: #205 volume support

### PR DESCRIPTION
**Pull request by ghost**
## Related issue

#205 Volume Support

## Proposed changes

The changes i made makes creating volumes in RunOptions possible. 

## Checklist

- [ x ] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md)
- [ x ] I have read the [security policy](../security/policy)
- [ x ] I confirm that this pull request does not address a security
      vulnerability. If this pull request addresses a security vulnerability, I
      confirm that I got green light (please contact
      [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push
      the changes.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation within the code base (if appropriate)

### CODE REVIEW

**Comment by aeneasr** [comment]
Thank you, this looks great! It looks like the CLA bot is not properly detecting your signature. To fix this, try the following:

```
$ git commit  --amend --author="Author Name <email@address.com>"
```

Ensure that `Author Name` is replaced with your GitHub username (e.g. `aeneasr`) and that the email address is replaced with the email address you have set up in GitHub (e.g. `3372410+aeneasr@users.noreply.github.com`):

```
$ git commit  --amend --author="aeneasr <3372410+aeneasr@users.noreply.github.com>"
```

Once that is done, you can force-push your changes (make sure to push to the correct remote and branch!):

```
$ git push --force <remote> HEAD:<branch>
```

File: docker/opts/opts_unix.go [hunk 1]
```diff
@@ -20,3 +20,12 @@ func MountParser(mount string) (source, destination string, err error) {
...
 	return "", "", errors.Wrap(fmt.Errorf("invalid mount format: got %s, expected <src>:<dst>", mount), "")
 }
+
+// VolumeParser parsed volume path.
+func VolumeParser(volume string) (source, target string, err error) {
+	st := strings.Split(volume, ":")
+	if len(st) == 3 {
+		return st[1], st[2], nil
+	}
+	return "", "", errors.Wrap(fmt.Errorf("invalid volume format: got %s, expected <drive>:<src>:<trgt>", volume), "")
```
Comment (colin-axner) [inline docker/opts/opts_unix.go:30]: Looks like this got copied from `opts_windows.go`. I think it should just take in `<src>:<dst>` (split with len 2)

It'd be great if we could allow the parser to accept `<src>:<dst>:ro` as a valid input and then pass `ro` as a bool indicating if the volume is read only

**Comment by aeneasr** [comment]
Closing due to inactivity

**Comment by mvid** [comment]
@aeneasr I have need for volumes as well, is there something I can do to push this over the finish line? Fork and fix?

**Pull request closed without merging**